Netlify deploy previews are failing to build

Recently our netlify deploy previews have been failing to build, example https://app.netlify.com/sites/etcd/deploys/647c5d107e85bc000792009c

Logs state:

```bash
9:45:30 PM: make: *** [Makefile:25: get-link-checker] Error 2
9:45:30 PM: ​
9:45:30 PM: "build.command" failed                                        
9:45:30 PM: ────────────────────────────────────────────────────────────────
9:45:30 PM: ​
9:45:30 PM:   Error message
9:45:30 PM:   Command failed with exit code 2: npm run build:preview (https://ntl.fyi/exit-code-2)
9:45:30 PM: ​
9:45:30 PM:   Error location
9:45:30 PM:   In build.command from netlify.toml:
9:45:30 PM:   npm run build:preview
9:45:30 PM: ​
9:45:30 PM:   Resolved config
9:45:30 PM:   build:
9:45:30 PM:     command: npm run build:preview
9:45:30 PM:     commandOrigin: config
9:45:30 PM:     environment:
9:45:30 PM:       - REVIEW_ID
9:45:30 PM:     publish: /opt/build/repo/public
9:45:30 PM:     publishOrigin: config
9:45:31 PM: Build failed due to a user error: Build script returned non-zero exit code: 2
9:45:31 PM: Failing build: Failed to build site
```

However when I run `npm install` followed by `npm run build:preview` the process completes sucessfully.

@nate-double-u - Any ideas what might be going on here?  If you can shed any light on what might be going wrong I'm happy to work on whatever fix is required 🙏🏻

--- a/Makefile
+++ b/Makefile
@@ -19,19 +19,6 @@ check-links: $(GET_LINK_CHECKER_IF_NEEDED)
 docker-serve:
 	docker run --rm -it -v $(PWD):/src -p 1313:1313 $(DOCKER_IMG) server $(DRAFT_ARGS)
 
-# Until htmltext >0.14.x is released, get and build our own from source:
 get-link-checker:
-	rm -Rf $(HTMLTEST_DIR)
-	mkdir -p $(HTMLTEST_DIR)/bin && \
-	cd $(HTMLTEST_DIR) && \
-	git clone --depth=1 https://github.com/wjdp/htmltest.git && \
-	( \
-		cd htmltest && \
-		./build.sh && \
-		cp bin/htmltest ../bin \
-	)
-
-# Once htmltext >0.14.x is released, replace the get-and-build code above with this:
-# get-link-checker:
-# 	rm -Rf $(HTMLTEST_DIR)/bin
-# 	curl https://htmltest.wjdp.uk | bash -s -- -b $(HTMLTEST_DIR)/bin
+	rm -Rf $(HTMLTEST_DIR)/bin
+	curl https://htmltest.wjdp.uk | bash -s -- -b $(HTMLTEST_DIR)/bin

